single.php

```diff
@@ -12,7 +12,7 @@
 
 		<?php while ( have_posts() ) : the_post(); ?>
 
-			<?php get_template_part( 'content', 'single' ); ?>
+			<?php get_template_part( 'content-single', get_post_format() ); ?>
```

Comment:
Twenty Fifteen and Twenty Fourteen do it this way:

```php
get_template_part( 'content', get_post_format() );
```
I'd prefer we use the latest way and not the way it was done in Twenty Eleven.